pandas/tests/frame/methods/test_replace.py

```diff
@@ -1403,3 +1403,28 @@ def test_replace_with_duplicate_columns(self, replacement):
         result["B"] = result["B"].replace(7, replacement)
 
         tm.assert_frame_equal(result, expected)
+
+    def test_replace_period_ignore_float(self):
+        """
+        Regression test for GH#34871: if df.replace(1.0, 0.0) is called on a df
+        with a Period column the old, faulty behavior is to raise TypeError
```

Comment:
you don't need to reference older versions like this, this commit would only be on master.

Reply:
thanks for the advise, removed that part!